src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -707,7 +707,7 @@ public CheckContextSummary ApplyGetContextSummary()
             var summary = new CheckContextSummary
             {
                 AllowsSideEffects = allowSideEffects,
-                IsPreV1Semantics = isV1,
+                IsPreV1Semantics = !isV1,

```

Comment:
>!isV1 [](http://example.com/codeflow?start=35&length=5)

Was this a bug?

Reply:
Yes, discussed with Mike and Hoa. The equality check above was always false before